cmd/av/stack.go

```diff
@@ -83,7 +85,65 @@ stack. This is useful for rebasing a whole stack on the latest changes from the
 base branch.
 `),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		return errors.New("unimplemented")
+		repo, err := getRepo()
+		if err != nil {
+			return err
+		}
+		originalBranch, err := repo.CurrentBranchName()
+		if err != nil {
+			return err
+		}
+
+		defer func() {
```

Comment:
I think for case of sync, let's also validate that there are no uncommitted changes before doing anything else.